### PR TITLE
Deleting users

### PR DESCRIPTION
**Pull request by Cheshiriks**

### CODE REVIEW

**Comment by akuleshov7** [comment]
What's about Project, Organization, Rating and Contests views? How the user will be shown there?

**Comment by Cheshiriks** [comment]
> What's about Project, Organization, Rating and Contests views? How the user will be shown there?

After deletion, its name is renamed to **_Deleted_** + ID

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsProfileMenuView.kt [hunk 1]
```diff
@@ -2,21 +2,40 @@ package com.saveourtool.save.frontend.components.views.usersettings
...
 import web.html.InputType
 
 @Suppress("MISSING_KDOC_TOP_LEVEL")
 class UserSettingsProfileMenuView : UserSettingsView() {
     private val card = cardComponent(isBordered = false, hasBg = true)
     @Suppress("TOO_LONG_FUNCTION", "LongMethod", "EMPTY_BLOCK_STRUCTURE_ERROR")
     override fun renderMenu(): VFC = VFC {
+        val deleteUserWindowOpenness = useWindowOpenness()
+
+        displayModal(
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsProfileMenuView.kt:23]: `window.confirm()`?

**Review by sanyavertolet — APPROVED** [review]

**Comment by akuleshov7** [comment]
> > What's about Project, Organization, Rating and Contests views? How the user will be shown there?
> 
> After deletion, its name is renamed to **_Deleted_** + ID

and they are not shown?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsProfileMenuView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsProfileMenuView.kt:23]: I want there to be a big, visual window here and people would think what they are doing :)

**Comment by kgevorkyan** [comment]
please make buttons more thin

**Review by akuleshov7 — APPROVED** [review]
Changes for the FE will be made in next PR

**Comment by Cheshiriks** [comment]
> please make buttons more thin

Fixed

**Pull request merged**